src/quo/components/avatars/wallet_user_avatar/view.cljs

```diff
@@ -0,0 +1,52 @@
+(ns quo.components.avatars.wallet-user-avatar.view
+  (:require [quo.components.avatars.wallet-user-avatar.style :as style]
+            [quo.components.markdown.text :as text]
+            [quo.theme :as quo.theme]
+            [react-native.core :as rn]
+            utils.string))
+
+(def properties
+  {:size-20 {:size        20
+             :font-size   :label
+             :font-weight :medium}
+   :size-24 {:size        24
+             :font-size   :label
+             :font-weight :semi-bold}
+   :size-32 {:size        32
+             :font-size   :paragraph-2
+             :font-weight :semi-bold}
+   :size-48 {:size        48
+             :font-size   :paragraph-1
+             :font-weight :semi-bold}
+   :size-64 {:size        64
+             :font-size   :paragraph-1
+             :font-weight :medium}
+   :size-80 {:size        80
+             :font-size   :heading-1
+             :font-weight :medium}})
+
+(def smallest-possible (first (keys properties)))
+(def biggest-possible (last (keys properties)))
+
+(defn- view-internal
+  "Options:
+
+  :full-name - string (default: nil) - used to generate initials
+  :customization-color  - keyword (default: nil) - color of the avatar
+  :size   - keyword (default: last element of properties object) - size of the
+  avatar
+  :monospace? - boolean (default: false) - use monospace font"
+  [{:keys [full-name customization-color size theme monospace?]
```

Comment:
@briansztamfater recently added `uppercase?` prop to this component btw, seems to be missing here 🤔

Reply:
ive omitted it intentionally, since it was only used by an initials retrieval code that we removed previously:

```
 f-name-initial (-> f-name
                           (#(if uppercase? (string/upper-case %) %))
                           (subs 0 1))
        l-name-initial (-> l-name
                           (#(if uppercase? (string/upper-case %) %))
                           (subs 0 1))
                           ```
                           
                           now its done by 
                           ```
                           (utils.string/get-initials full-name (if small? 1 2))]]))
                           ```

Reply:
Great 👌